package.json

```diff
@@ -20,9 +20,9 @@
     "node": "^18 || ^20"
   },
   "dependencies": {
-    "@alaskaairux/icons": "^4.36.2",
-    "@aurodesignsystem/auro-dropdown": "^2.10.2",
-    "@aurodesignsystem/auro-input": "^2.17.4",
+    "@alaskaairux/icons": "^4.38.0",
+    "@aurodesignsystem/auro-dropdown": "^2.10.5",
+    "@aurodesignsystem/auro-input": "^2.17.6",
```

Comment:
Just so that everyone is 100% understanding what is happening here. 

This update links way back to an assumption that just because one element has a dependency on an element with a version and something else will also have a dependency on that same element but at a different version, this will cause things to work inconsistently. 

Node packages DO NOT work that way. To keep from repeating a VERY long investigation, please see https://github.com/AlaskaAirlines/WC-Generator/discussions/505#discussioncomment-8550246

What is most likely happening is that consumers are NOT updating all their dependencies the same way. A process of cherry-picking updates will lend itself to misalignment. Additionally what version gets used in the browser depends on what gets registered in the browser first. This is why many devs complained of inconsistent behavior locally. 

What is HIGHLY recommended is that all devs perform a `npm ci` locally allowing npm to dedupe all shared dependencies and in conjunction with the `^` operator will install the latest version below the next MAJOR version. 

The only way to ensure that a version is installed and registered with the browser is to customize the name via the dependent element.